conf/ocsci/ocs-converged-qe-addon.yaml

```diff
@@ -0,0 +1,3 @@
+---
+ENV_DATA:
+  addon_name: "ocs-converged-qe"
```

Comment:
May be we can add  this config in `conf/deployment/rosa/` directory similar to  
https://github.com/red-hat-storage/ocs-ci/blob/7ed2885b70642980db04905deac12972e5b8550f/conf/deployment/rosa/managed_1az_converged_3m_3w_m52x.yaml

Reply:
Added managed_1az_converged_qe_3m_3w_m52x.yaml configuration

Reply:
The PR is open for a long time and when I started it, the QE addon was relevant. Right now we probably don't need the configuration for our testing (but I think that we can keep it for now, if you want the configuration gone then we can remove it from this PR and just merge changes to the framework - the configuration was provided for purpose of verifying the change) but we need other functionality from this PR that allows us work with custom addon names.